capstone/src/main/java/com/google/step/clients/MetricsClient.java

```diff
@@ -116,125 +99,57 @@ public MetricsClient() {}
         PreparedQuery preparedQuery = dataStore.prepare(query);
         List<Entity> results = preparedQuery.asList(FetchOptions.Builder.withDefaults());
 
-        List<Map<String, Object>> resultList = new ArrayList<>();
+        List<WeeklyPageView> pageViewsList = new ArrayList<>();
 
-        Map<String, Object> map;
+        WeeklyPageView pageView;
```

Comment:
This is unnecessary. In this case, prefer `pageViewsList.add(new WeeklyPageView(...));`

Reply:
Good catch. Done.